README.md

```diff
@@ -44,20 +44,22 @@ Configuration can be provided via:
+| TESTRAIL_BRANCH_ENV           | branchEnv          | Which ENV variable is used to store branch name on which tests are run.                                                                                                                                                                                                                                                |          `BRANCH`           |          |
+| TESTRAIL_BUILD_NO_ENV         | buildNoEnv         | Which ENV variable is used to store build number of tests run.                                                                                                                                                                                                                                                         |       `BUILD_NUMBER`        |          |
+| TESTRAIL_DATE_FORMAT          | dateFormat         | What date format should be used for `%DATE%` placeholder. https://momentjs.com/ formats supported.                                                                                                                                                                                                                     |    `YYYY-MM-DD HH:mm:ss`    |          |
+| TESTRAIL_RUN_CLOSE_AFTER_DAYS | runCloseAfterDays  | After how many days should reporter close old Runs in testrail.                                                                                                                                                                                                                                                        |                             |          |
+| TESTRAIL_UPLOAD_SCREENSHOTS   | uploadScreenshots  | Should upload screenshots to testrail. Requires test result edit enabled in testrail.                                                                                                                                                                                                                                  |           `false`           |          |
+| TESTRAIL_RUN_ID               | runId              | Run id which test cases are stored. Ex `R123`                                                                                                                                                                                                                                                                          |                             |          |
+| TESTRAIL_RUN_CREATED_MANUALLY | runCreatedManually | Tells to the reporting tool to no create or update the given test run, you need to create the mapping for the testcases manually                                                                                                                                                                                       |           `false`           |          |
```

Comment:
Hmm, I think that this flag does not provide correct intent.
Maybe naming it along the lines of  **TESTRAIL_UPDATE_RUN_TEST_CASES** and defaulting it to **true** would be more intuitive.